### PR TITLE
feat(data model): update according to new data model

### PR DESCRIPTION
**Pull request by shimoncohen**

### CODE REVIEW

File: src/common/i18n/en.json [hunk 1]
```diff
@@ -40,7 +40,8 @@
   "export-table.nodata": "No Exported packages to show",
   "export-table.table-column-header.fileName.text": "File Name",
   "export-table.table-column-header.sizeEst.text": "Est Size(Mb)",
-  "export-table.table-column-header.tilesEst.text": "Est Tiles",
+  "export-table.table-column-header.realSize.text": "Real Size(Mb)",
+  "export-table.table-column-header.expirationTime.text": "Epiration date",
```
Comment (asafMasa) [inline src/common/i18n/en.json:44]: spelling

File: src/__mocks-data__/exportedPackages.ts [hunk 2]
```diff
@@ -1,17 +1,17 @@
-import { ExportStatus } from "../exporter/models/exportTaskStatus";
+import { ExportStatus } from '../exporter/models/exportTaskStatus';
 
 const MOCK_EXPORTED_PACKAGES = [
   {
     fileName: 'kuku',
-    sizeEst: 23,
-    tilesEst: 120,
+    realSize: 23,
```
Comment (asafMasa) [inline src/__mocks-data__/exportedPackages.ts:6]: revert sizeEst

File: src/__mocks-data__/exportedPackages.ts [hunk 3]
```diff
@@ -24,15 +24,15 @@ const MOCK_EXPORTED_PACKAGES = [
   },
   {
     fileName: 'muku',
-    sizeEst: 345,
-    tilesEst: 2000,
+    realSize: 345,
```
Comment (asafMasa) [inline src/__mocks-data__/exportedPackages.ts:27]: revert sizeEst

File: src/exporter/components/export-table/export-table-dialog.tsx [hunk 4]
```diff
@@ -147,10 +147,22 @@ export const ExportSatusTableDialog: React.FC<ExportSatusTableDialogProps> = obs
         },
         {
           headerName: intl.formatMessage({
-            id: 'export-table.table-column-header.tilesEst.text',
+            id: 'export-table.table-column-header.realSize.text',
```
Comment (asafMasa) [inline src/exporter/components/export-table/export-table-dialog.tsx:150]: revert tile size estimation

File: src/exporter/models/exportTaskStatus.ts [hunk 5]
```diff
@@ -20,16 +21,16 @@ const bbox = types.model({
 
 export const exportTaskStatus = types.model({
   fileName: types.string,
-  sizeEst: types.number,
-  tilesEst: types.number,
+  realSize: types.number,
```
Comment (asafMasa) [inline src/exporter/models/exportTaskStatus.ts:24]: sizeEst revert

File: src/exporter/models/exporterStore.spec.ts [hunk 6]
```diff
@@ -35,8 +35,7 @@ describe('Exporter Store', () => {
 
     await exporterStore.startExportGeoPackage({
       packName: 'kuku',
-      sizeEst: 20000,
-      tilesEst: 200,
+      realSize: 20000,
```
Comment (asafMasa) [inline src/exporter/models/exporterStore.spec.ts:38]: revert sizeEst

File: src/exporter/models/exporterStore.spec.ts [hunk 7]
```diff
@@ -57,8 +56,7 @@ describe('Exporter Store', () => {
...
```
Comment (asafMasa) [inline src/exporter/models/exporterStore.spec.ts:59]: revert sizeEst

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/exporter/components/export-table/export-table-dialog.tsx (hunk 4 above)
Comment (shimoncohen) [reply src/exporter/components/export-table/export-table-dialog.tsx:160]: Wasn't tile size removed?

**Review by asafMasa — APPROVED** [review]

**Pull request merged**